### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by gp201**
This PR adds a README file for the organization.

Markdown Render: https://github.com/gp201/.github/blob/add-readme/profile/README.md

image

### CODE REVIEW

**Comment by gp201** [comment]
@seanlip PTAL

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Note: Oppiabot is not accepting contributors right now.
+
+## Connect with us!
+
+[<img height="30" src="https://img.shields.io/badge/twitter-1DA1F2.svg?&style=for-the-badge&logo=twitter&logoColor=white" />][twitter] [<img height="30" src="https://img.shields.io/badge/linkedin-0077B5.svg?&style=for-the-badge&logo=linkedin&logoColor=white" />][LinkedIn] [<img height="30" src = "https://img.shields.io/badge/facebook-1877F2.svg?&style=for-the-badge&logo=facebook&logoColor=white">][Facebook] [<img height="30" src = "https://img.shields.io/badge/medium-12100E.svg?&style=for-the-badge&logo=medium&logoColor=white">][medium] [<img height="30" src = "https://img.shields.io/badge/oppia.org%20youtube-FF0000.svg?&style=for-the-badge&logo=youtube&logoColor=white">][oppia-org-youtube] [<img height="30" src = "https://img.shields.io/badge/oppia%20dev%20youtube-FF0000.svg?&style=for-the-badge&logo=youtube&logoColor=white">][dev-youtube]
+
+[twitter]: https://twitter.com/oppiaorg
+[linkedIn]: https://www.linkedin.com/company/oppia-org/
+[medium]: https://medium.com/@oppia.org
```
Comment (seanlip) [inline profile/README.md]: This link is wrong, please use this one: https://medium.com/oppia-org

Otherwise, looks good, thanks!

**Review by seanlip — CHANGES_REQUESTED** [review]
Hi @gp201, just one comment. Thanks!

File: profile/README.md (hunk 1 above)
Comment (gp201) [reply profile/README.md]: Done

**Comment by gp201** [comment]
@seanlip PTAL

**Review by seanlip — APPROVED** [review]
Thanks, @gp201 -- LGTM!

**Pull request merged**